README.md

```diff
@@ -1 +1,157 @@
-# Metrics-webapp
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Stock Screaner <a name="about-project"></a>
+
+**Stock Screaner** is a tool quickly find current stock prices in USA companies...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Build With</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://redux-toolkit.js.org/">Redux toolkit</a></li>
+    <li><a href="https://site.financialmodelingprep.com/developer/docs/">Financial Modeling Prep API</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Filter companies by Company Names**
+- **Display all companies by Company Name and then filter by name or symbol**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ - Live Demo: [netlify](https://superb-puppy-322d9f.netlify.app/)
+
+ **Loom Video Link**
+[Description of project](https://www.loom.com/share/7d89819ba3d94f64ac37fb8c3e12494f)
```

Comment:
- [ ] Good video presentation, however, it's rather too long than expected. As per the requirements for video presentation, let keep the video length to a maximum of **5 minutes**.